lib/perl/Genome/Model.pm

```diff
@@ -471,6 +471,8 @@ sub __display_name__ {
 sub create {
     my $class = shift;
 
+    no warnings 'redefine';
+    local *__errors__ = sub { () };
```

Comment:
If we're pretending to the caller that there's not a transaction here, would it make sense to override `UR::Context::Transaction->changes_can_be_saved` instead of picking and choosing which errors to ignore?  (Or, would it be overkill to introduce an option for software transactions to not check validity of objects before promoting them to the next context?)